src/frontend/apps/desk/src/features/teams/components/PanelTeam.tsx

```diff
@@ -15,47 +16,80 @@ interface TeamProps {
 export const PanelTeam = ({ team }: TeamProps) => {
   const { t } = useTranslation();
   const { colorsTokens } = useCunninghamTheme();
+  const {
+    query: { id },
+  } = useRouter();
 
   // There is at least 1 owner in the team
   const hasMembers = team.accesses.length > 1;
+  const isActive = team.id === id;
 
   const commonProps = {
     className: 'p-t',
     width: 52,
     style: {
       borderRadius: '10px',
       flexShrink: 0,
+      background: '#fff',
     },
   };
 
+  const activeStyle = `
+    border-right: 4px solid ${colorsTokens()['primary-600']};
+    background: ${colorsTokens()['primary-400']};
+    span{
+      color: ${colorsTokens()['primary-text']};
+    }
+  `;
+
+  const hoverStyle = `
+    &:hover{
+      border-right: 4px solid ${colorsTokens()['primary-400']};
+      background: ${colorsTokens()['primary-300']};
+      
+      span{
+        color: ${colorsTokens()['primary-text']};
+      }
+    }
+  `;
+
   return (
-    <Box as="li">
-      <StyledLink href={`/teams/${team.id}`}>
-        <Box $align="center" $direction="row" $gap="0.5rem">
-          {hasMembers ? (
-            <IconGroup
-              aria-label={t(`Teams icon`)}
-              color={colorsTokens()['primary-500']}
-              {...commonProps}
-              style={{
-                ...commonProps.style,
-                border: `1px solid ${colorsTokens()['primary-300']}`,
-              }}
-            />
-          ) : (
-            <IconNone
-              aria-label={t(`Empty teams icon`)}
-              color={colorsTokens()['greyscale-500']}
-              {...commonProps}
-              style={{
-                ...commonProps.style,
-                border: `1px solid ${colorsTokens()['greyscale-300']}`,
-              }}
-            />
-          )}
-          <Text $weight="bold">{team.name}</Text>
-        </Box>
-      </StyledLink>
+    <Box
+      as="li"
+      $css={`
+        transition: all 0.2s ease-in; 
+        border-right: 4px solid transparent;
+        ${isActive ? activeStyle : hoverStyle}
+      `}
+    >
```

Comment:
We should ask Johann how to style an active when hovering it. It feels weird to have no changing in style when hovering a selected item.